JavaScript/JavaScriptSDK.Tests/CheckinTests/Logging.tests.ts

```diff
@@ -165,7 +166,36 @@ class LoggingTests extends TestClass {
                     Assert.equal("AI (Internal): " + "NONUSRACT_BrowserCannotReadLocalStorage message:\"error!\"", InternalLogging.queue[0].message);
 
                     // cleanup
-                    
+                    InternalLogging.verboseLogging = () => false;
```

Comment:
It's a class with a bunch of static methods and properties. It's hard to unit tests it :angry: 

...anyway, I changed it slightly to make the code more clear and easier to read.